### PR TITLE
feat: added a graphql scheme to generate sample hooks

### PR DESCRIPTION
**Pull request by k3nt0w**
## Proposed Changes

- Got the scheme from `hasura` and saved it.
- Added an `allocator_allocation_result` query as a sample.
- Auto-generated hooks from the query.

## Implementation

- Got the scheme and ran `codegen` .

## Remaining Task

- [x] Specify the correct API HOST

## ⚠️ NOTE

Because the GraphQL server did not correspond to CORS, I checked the operation after stopping CORS with **Safari** .

### CODE REVIEW

File: packages/web/src/fixtures/withApollo/index.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const isBrowser = typeof window !== 'undefined'
 export default withApollo<NormalizedCacheObject>(
   ({ ctx, headers }) => {
     const withHttp = createHttpLink({
-      uri: `${process.env.API_ORIGIN}/graphql`, // Server URL (must be absolute),
+      uri: `${process.env.END_POINT}`, // Server URL (must be absolute),
```
Comment (k3nt0w) [inline packages/web/src/fixtures/withApollo/index.tsx:25]: I found the following description in the [official document](https://www.apollographql.com/docs/link/links/http/#context), so I named it this way.

> uri: a string of the endpoint you want to fetch from

File: packages/web/src/pages/about.tsx [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
   return (
     <div>
       <Button onClick={() => Router.push('/')}>please click here!</Button>
       {loading && <div>loading.......</div>}
       {data && (
         <div>
-          {data.list.map(user => (
-            <li key={user.id}>{user.name}</li>
+          {data.allocator_allocation_result.map(d => (
+            <li key={d.event_id}>{d.result}</li>
```
Comment (k3nt0w) [inline packages/web/src/pages/about.tsx:20]: This is just sample code. The result is like this screenshots and we can confirm good working apollo client.

スクリーンショット 2020-04-10 2 46 20

**Comment by abyssparanoia** [comment]
@k3nt0w 
Thunk you for your PR.

Please remove backend pacakge, because we also don't need sample GraphQL endpoint?

File: packages/web/src/pages/about.tsx (hunk 2 above)
Comment (abyssparanoia) [reply packages/web/src/pages/about.tsx:20]: 💯

**Comment by k3nt0w** [comment]
@abyssparanoia 
I was going to do that job in my next PR. What should be done with this PR?

**Comment by abyssparanoia** [comment]
Ok

**Pull request merged**